src/sdk/api/fetchCompositeChartData.js

```diff
@@ -44,11 +44,18 @@ const getAggregations = chart => {
 }
 
 const getCompositeChartPayload = chart => {
-  const { context } = chart.getMetadata()
-  const { nodeIds, dimensions, postAggregationMethod, filteredLabels } = chart.getAttributes()
+  const { context, nodeIDs: nodeIdsThatContainCharts } = chart.getMetadata()
```

Comment:
why not `chartNodeIds`?

Reply:
Hmm I think `nodeIdsThatContainChart` is more explanatory.